### PR TITLE
add Reproducible Builds badge

### PR DESCRIPTION
**Pull request by hboutemy**
congratulations, your latest releases were proven reproducible
Let's celebrate with this badge :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ loves Error Prone: producing high-quality and consistent Java
 code_][picnic-blog-ep-post].
 
 [![Maven Central][maven-central-badge]][maven-central-search]
+[![Reproducible Builds][reproducible-builds-badge]][reproducible-builds-link]
```
Comment (Stephan202) [inline README.md:23]:
```suggestion
[![Reproducible Builds][reproducible-builds-badge]][reproducible-builds-report]
```

File: README.md [hunk 2]
```diff
@@ -226,3 +227,5 @@ guidelines][contributing].
 [refaster]: https://errorprone.info/docs/refaster
 [refaster-rules-bigdecimal]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BigDecimalRules.java
 [refaster-rules]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/
+[reproducible-builds-badge]: https://img.shields.io/badge/Reproducible_Builds-ok-success?labelColor=1e5b96
+[reproducible-builds-link]: https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/tech/picnic/error-prone-support/error-prone-support/README.md
```
Comment (Stephan202) [inline README.md:231]:
```suggestion
[reproducible-builds-report]: https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/tech/picnic/error-prone-support/error-prone-support/README.md
```

File: README.md [hunk 3]
```diff
@@ -226,3 +227,5 @@ guidelines][contributing].
 [refaster]: https://errorprone.info/docs/refaster
 [refaster-rules-bigdecimal]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/BigDecimalRules.java
 [refaster-rules]: https://github.com/PicnicSupermarket/error-prone-support/blob/master/error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/
+[reproducible-builds-badge]: https://img.shields.io/badge/Reproducible_Builds-ok-success?labelColor=1e5b96
```
Comment (Stephan202) [inline README.md:230]: How was the color selected? :smile:

(I guess that eventually it'd be cool to have dynamic badges that go red if the latest release is no longer reproducible :sparkles:.)

**Review by Stephan202 — APPROVED** [review]
Thanks! Looks very nice! :tada:

image

Made one suggestion around link naming :)

Suggested commit message:
```
Add Reproducible Builds badge to README (#333)
```

NB: I didn't forget about jvm-repo-rebuild/reproducible-central#93!

**Review by rickie — APPROVED** [review]
Nice contribution @hboutemy! 🚀 

Changes, @Stephan202's suggestion, and the suggested commit message LGTM!

File: README.md (hunk 3 above)
Comment (hboutemy) [reply README.md:230]: color was chosen after Reproducible Builds logo

**Pull request merged**